examples/gallery/plot/legend.py

```diff
@@ -0,0 +1,26 @@
+"""
+Legend
+------
+
+The :meth:`pygmt.Figure.legend` method can automatically create a legend for
+symbols plotted using :meth:`pygmt.Figure.plot`.
```

Comment:
Might want to be a little bit more descriptive? E.g. tell users that they will need to add `label="SomeText"` to the `plot` argument in order for `legend` to work!